auditbeat/module/auditd/_meta/fields.yml

```diff
@@ -821,6 +824,25 @@
         - name: removed
           type: keyword
           description: number of deleted files
+        - name: socket
+          type: group
+          fields:
+          - name: port
+            type: keyword
+            description: The port number.
+          - name: saddr
+            type: keyword
+            description: The raw socket address structure.
+          - name: addr
```

Comment:
I suggest not to use abbreviation for address.

Reply:
I opted for `addr` because that's the exact name coming from the kernel. But I can rename this is `go-libaudit`.